core/src/test/scala/org/apache/spark/sql/delta/DelegatingLogStoreSuite.scala

```diff
@@ -101,25 +105,47 @@ class DelegatingLogStoreSuite
 
   test("DelegatingLogStore resolution using default scheme confs") {
     for (scheme <- DelegatingLogStore.s3Schemes) {
-      testDelegatingLogStore(scheme, None, DelegatingLogStore.defaultS3LogStoreClassName, false)
+      testDelegatingLogStore(
+        scheme,
+        schemeConf = None,
+        expClassName = DelegatingLogStore.defaultS3LogStoreClassName,
+        expAdaptor = true)
     }
     for (scheme <- DelegatingLogStore.azureSchemes) {
-      testDelegatingLogStore(scheme, None, DelegatingLogStore.defaultAzureLogStoreClassName, false)
+      testDelegatingLogStore(
+        scheme,
+        schemeConf = None,
+        expClassName = DelegatingLogStore.defaultAzureLogStoreClassName,
+        expAdaptor = true)
     }
-    testDelegatingLogStore(fakeSchemeWithNoDefault, None,
-      DelegatingLogStore.defaultHDFSLogStoreClassName, false)
+    testDelegatingLogStore(
+      scheme = fakeSchemeWithNoDefault,
+      schemeConf = None,
+      expClassName = DelegatingLogStore.defaultHDFSLogStoreClassName,
+      expAdaptor = true)
   }
 
   test("DelegatingLogStore resolution using customized scheme confs") {
     val allTestSchemes = DelegatingLogStore.s3Schemes ++ DelegatingLogStore.azureSchemes +
       fakeSchemeWithNoDefault
     for (scheme <- allTestSchemes) {
       for (store <- Seq(
+        // default (java) classes (in io.delta.storage)
         DelegatingLogStore.defaultS3LogStoreClassName,
         DelegatingLogStore.defaultAzureLogStoreClassName,
         DelegatingLogStore.defaultHDFSLogStoreClassName,
+        // deprecated (scala) classes
+        classOf[org.apache.spark.sql.delta.storage.S3SingleDriverLogStore].getName,
+        classOf[org.apache.spark.sql.delta.storage.AzureLogStore].getName,
+        classOf[org.apache.spark.sql.delta.storage.HDFSLogStore].getName,
         customLogStoreClassName)) {
-        testDelegatingLogStore(scheme, Some(store), store, store == customLogStoreClassName)
+
+        // we set spark.delta.logStore.${scheme}.impl -> $store
+        testDelegatingLogStore(
+          scheme,
+          schemeConf = Some(store),
+          expClassName = store,
+          expAdaptor = store.contains("io.delta.storage") || store == customLogStoreClassName)
```

Comment:
nit: indention

Reply:
@allisonport-db this indentation actually seems right to me (2 space indent). what looks wrong to you?